### PR TITLE
README Minor tweaks

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,16 +77,14 @@ docker run -it --rm -p 1331:1331 -v ./nginx.conf:/etc/nginx/nginx.conf:ro --name
...
 - run the following command `npm run dev`. This will host a dev server on port 5174 and start watching for changes in `/angular` directory
 
 > [!NOTE]
 > For compatibility with Node.js versions 17 and higher, it's essential to set the `NODE_OPTIONS` environment variable to `--openssl-legacy-provider` when loading and running the angularJS app. This configuration is specified in the `.npmrc` file located at the root of the Angular folder `ServicePulse\src\ServicePulse.Host\angular`. Any npm command executed from the Angular directory, including those in CI/Release workflows, will automatically have the necessary environment variable set.
-
-_Note: If you receive an error similar to `Error: error:0308010C:digital envelope routines::unsupported` when Webpack attempts to execute `LoadRunner.js`, it means that you need to configure Node to use a legacy SSL provider. You can do this by setting an environment variable called `NODE_OPTIONS` to the value `--openssl-legacy-provider`. E.g. Run `export NODE_OPTIONS=--openssl-legacy-provider` from a bash shell.
+> On *nix environment, run the following command from a bash shell: `export NODE_OPTIONS=--openssl-legacy-provider`.
```
Comment (johnsimons) [inline README.md:87]: It seems this is being done automatically as part of `.npmrc`, so I would remove this statement entirely.

Comment (mauroservienti) [reply README.md:87]: I'll double-check on my macOS box; I think I had to do it.

Comment (johnsimons) [reply README.md:87]: I have already tested it on my Mac 😉

**Review by johnsimons — APPROVED** [review]
Makes sense, but also, this step hopefully will be removed very soon

**Pull request merged**